system/jlib/jptree.cpp

```diff
@@ -6635,13 +6635,18 @@ class CJSONReader : public CJSONReaderBase<X>, implements IPTreeReader
                 iEvent->newAttribute(name, value.str());
             return;
         }
-        else if ('#'==*name)
+        else if ('#'==*name) // binary
```

Comment:
comment is a bit misleading, could be binary and/or mixed content.

Reply:
yes, a hangover from previous version, I'll remove.